pkg/storage/gcp/saver.go

```diff
@@ -116,7 +116,7 @@ func (s *Storage) Save(ctx context.Context, module, version string, mod, zip, in
 // upload waits for either writeToBucket to complete or the context to expire
 func upload(ctx context.Context, errs chan<- error, bkt *storage.BucketHandle, module, version, ext string, file []byte) {
 	select {
-	case errs <- writeToBucket(ctx, bkt, key(module, version, ext), file):
+	case errs <- writeToBucket(ctx, bkt, fmt.Sprintf("%s/@v/%s.%s", module, version, ext), file):
```

Comment:
sweet

Reply:
i like your proactive conflict avoidance :-) in a good way

Reply:
lol thanks